release notes/v0.46.0.md

```diff
@@ -0,0 +1,131 @@
+k6 `v0.46` is here 🎉! This release includes:
+
+- (_optional_) `<highlight of breaking changes>`
+- `<Summary of new features>` (_one or multiple bullets_)
+
+
+## Breaking changes
+
+- `#pr`, `<small_break_1>`
+- `#pr`, `<small_break_2>`
+
+### (_optional h3_) `<big_breaking_change>` `#pr`
+
+## New features
+
+### Loki log output sending additional headers [#3227](https://github.com/grafana/k6/pull/3227)
```

Comment:
I couldn't find a corresponding PR in the docs for this, do we need to add it in this section? https://k6.io/docs/using-k6/k6-options/reference/#log-output

Reply:
Yes; my bad, I'll get on it

Reply:
Thank you 🙇🏻

Reply:
Created: https://github.com/grafana/k6-docs/pull/1293